src/readme.html

```diff
@@ -66,33 +66,33 @@ <h3>Recommendations</h3>
 </ul>
 
 <h2>Online Resources</h2>
-<p>If you have any questions that aren&#8217;t addressed in this document, please take advantage of WordPress&#8217; numerous online resources:</p>
+<p>If you have any questions that aren&#8217;t addressed in this document, please take advantage of ClassicPress&#8217; numerous online resources:</p>
```

Comment:
"please take advantage of ClassicPress' numerous online resources" - this doesn't make sense.

Let's undo all of the changes to `src/readme.html` for now, and address this separately in #33.  I wonder if we even need this file at all, honestly.